subprojects/docs/src/docs/release/notes.md

```diff
@@ -30,6 +30,12 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 
 TBD - load from cache performance improvements and reduced memory consumption for Android builds
 
+## Improved name matching
+
+Previously, if you wanted to execute a task with abbreviated names, you needed to use camel case naming. For example, the command `./gradlew mAP:help` worked if the target project name was `myAweSomeProject`. Now, the name matching works with kamel-case names: the command above works even if you name your project `my-awesome-project`.
```

Comment:
`the name matching works` -> `the name matching also works`

Otherwise it sounds like we _replaced_ the old with the new thing.

Reply:
@jjohannes I've rewritten the release notes section, please have another look.